staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -5175,8 +5175,27 @@ type SecurityContext struct {
 	// 2) has CAP_SYS_ADMIN
 	// +optional
 	AllowPrivilegeEscalation *bool `json:"allowPrivilegeEscalation,omitempty" protobuf:"varint,7,opt,name=allowPrivilegeEscalation"`
+	// ProcMount denotes the type of proc mount to use for the containers.
```

Comment:
This should be lower cased when referenced in external API godoc (since that shows up in openapi and is displayed in tools where users only see the JSON field).

Reply:
the other ones above this were not lowercased so just wondering what you mean exactly :)

Reply:
Basically the original API we didn't enforce it and we started doing it.  If you look at newer APIs we enforce that consistently.

Reply:
gotcha will update